lms/static/sass/bootstrap/_legacy.scss

```diff
@@ -109,3 +108,44 @@ $shadow-l1: rgba(0, 0, 0, 0.1) !default;
 $shadow-l2: rgba(0, 0, 0, 0.05) !default;
 $shadow-d1: rgba(0, 0, 0, 0.4) !default;
 $shadow-d2: rgba($black, 0.6) !default;
+
+// Blues
+$blue-d: rgba(6, 86, 131, 100) !default;
+$blue-d1: mix($black, $blue, 20%) !default;
+$blue-d3: desaturate($blue, 15%) !default;
+$blue-b: rgba(0, 117, 180, 100) !default;
+$blue-a: rgba(41, 145, 195, 100) !default;
```

Comment:
Is there a reason to prefer using the name `blue`? The pattern library and bootstrap both use the notion of `primary`. My initial take is that we should prefer:
```
$primary-base: rgba(0, 117, 180, 100) !default;
$primary-accent: rgba(41, 145, 195, 100) !default;
```
Any reason this is less desirable?

Reply:
No, there is no reason. I was not sure how to define these new variables, are rules to do it? 
For example, a color using the palette function, should I define the new variable with property: gray-base, gray-accent?
I did not find a similar function on bootstrap or edx that would generate the colors used with the pattern library. So i defined these news blues and grays variables.

Reply:
I think we should aim to map (as best as we can) to bootstrap's color concepts:

```
primary
secondary
success
info
warning
danger
light
dark
```

See the [bootstrap colors src here](https://github.com/twbs/bootstrap/blob/v4.4.1/scss/_variables.scss#L69). Bootstrap has semantic color variable names similar to the pattern library (see the [colors src in pattern library](https://github.com/edx/ux-pattern-library/blob/cfee8ee7d088f1067571e5a195f1c98198412cb3/pattern-library/sass/global/_settings.scss#L74)). Bootstrap version 4 does not have defined shades of these colors like Pattern library does so the variables you've added make sense to me. I'd just change the variables names to map to the semantic names a little better.

Anyone building an open edx theme should not find themselves writing things like
```
$blue-b: red;
```
The one exception to these "semantic" variable names I think is `gray` which we can leave alone. Roughly I think we should end up with variable names like:

```
$primary-dark:       rgba(6, 86, 131, 100);
$primary-base:       rgba(0, 117, 180, 100);
$primary-accent:     rgba(41, 145, 195, 100);
$primary-back:       rgba(204, 227, 240, 100);
$primary-x-back:     rgba(242, 248, 251, 100);

$gray-dark:       rgba(17, 17, 17, 100);
$gray-base:       rgba(65, 65, 65, 100);
$gray-accent:     rgba(160, 160, 160, 100);
$gray-back:       rgba(217, 217, 217, 100);
$gray-x-back:     rgba(245, 245, 245, 100);
```